02-barao/outras_necessidades.tex

```diff
@@ -182,6 +182,32 @@ \subsection{Bancos}
   Barão.
 \end{itemize}
 
+\subsection{Caixas Eletrônicos}
+Além das agências mencionadas na seção anterior, existem os seguintes caixas em
+Barão:
+
+\begin{itemize}
+\item \textbf{Dalben}
+  \\Endereço: Av. Albino J. B. de Oliveira, 511
+  \\Horário: Segunda à Sábado das 7:00 às 23:00, Domingo das 7:00 às 22:00
+  \\
+  \\Dentro do mercado.
+
+\item \textbf{Pague Menos}
+  \\Endereço: Rua Benedito Alves Aranha, 130
+  \\Horário: Segunda à Domingo das 8:00 às 22:00
+  \\
+  \\No estacionamento atrás do mercado.
+
+\item \textbf{Maria Gasolina}
+  \\Endereço: Av. Albino J. B. de Oliveira, 1480
+  \\Horário: Segunda à Domingo das 6:00 às 0:00
+  \\
+  \\No posto do final da Av. 2 (Atílio Martini), dentro da loja de 
```

Comment:
Talvez comentar que fica do lado do McDonalds ajude mais as pessoas. Eu faço isso quando for revisar com o manual compilado